components/eam/src/physics/cam/cam_diagnostics.F90

```diff
@@ -602,27 +606,34 @@ subroutine diag_init()
 
    ! outfld calls in diag_surf
 
-   call addfld ('SHFLX',horiz_only,    'A','W/m2','Surface sensible heat flux')
-   call addfld ('LHFLX',horiz_only,    'A','W/m2','Surface latent heat flux')
-   call addfld ('QFLX',horiz_only,    'A','kg/m2/s','Surface water flux')
+   call addfld ('SHFLX',horiz_only,    'A','W/m2','Surface sensible heat flux', &
+   standard_name='surface_upward_sensible_heat_flux')
+   call addfld ('LHFLX',horiz_only,    'A','W/m2','Surface latent heat flux', &
+   standard_name = 'surface_upward_latent_heat_flux')
+   call addfld ('QFLX',horiz_only,    'A','kg/m2/s','Surface water flux', &
+   standard_name = 'water_evapotranspiration_flux')
 
    call addfld ('TAUX',horiz_only,    'A','N/m2','Zonal surface stress')
    call addfld ('TAUY',horiz_only,    'A','N/m2','Meridional surface stress')
-   call addfld ('TREFHT',horiz_only,    'A','K','Reference height temperature')
+   call addfld ('TREFHT',horiz_only,    'A','K','Reference height temperature', &
+      standard_name='air_temperature')
```

Comment:
Surprised the standard name is not "surface_air_temperature". Probably we just have to take "air_temperature" because that is the only one suitable.